waspc/data/Generator/templates/sdk/wasp/client/operations/internal/index.ts

```diff
@@ -1,19 +1,16 @@
 import { api, handleApiError } from 'wasp/client/api'
 import { HttpMethod } from 'wasp/client'
-import {
-  serialize as superjsonSerialize,
-  deserialize as superjsonDeserialize,
- } from 'superjson'
+import { serialize, deserialize } from 'wasp/core/serialization'
 
 // PRIVATE API
 export type OperationRoute = { method: HttpMethod.Post, path: string }
 
 // PRIVATE API
 export async function callOperation(operationRoute: OperationRoute, args: any) {
   try {
-    const superjsonArgs = superjsonSerialize(args)
+    const superjsonArgs = serialize(args)
```

Comment:
Let's change the variable name too (to completely decouple the code of SuperJson).